requirements/edx/base.txt

```diff
@@ -147,6 +147,7 @@ selenium==2.42.1
 splinter==0.5.4
 testtools==0.9.34
 
+
```

Comment:
I had used `testfixtures==4.1.2` , and removed it later. 
Removing the line no 150